include/faker-cxx/System.h

```diff
@@ -46,6 +46,19 @@ class System
      */
     static std::string fileExt(const std::optional<std::string>& mimeType = std::nullopt);
 
+    /**
+     * @brief Returns a file extension.
+     *
+     * @param mimeType value of MimeType enum.
+     *
+     * @returns A file extension.
+     *
+     * @code
+     * System::fileExt(MimeType::Image) // "png"
+     * @endcode
+     */
+    static std::string fileExt(MimeType mimeType);
```

Comment:
please add mimeType as optional parameter and if not provided return random from all file extensions